setup.py

```diff
@@ -32,7 +32,7 @@
         'pyasn1>=0.0.13',
```

Comment:
I.. would rather not, because this is still installed system-wide in the old way for prod.